### PR TITLE
Ipinfo Batch

### PR DESCRIPTION
**Pull request by avidan-H**
## Status
In Hold(batch API endpoint untested - we don't have a paid plan)

## Description
Adding batch support for ipinfo's `ip` command. Related to this [issue](https://github.com/demisto/etc/issues/19495).

## Required version of Demisto
any

## Does it break backward compatibility?
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Dependencies
Mention the dependencies of the entity you changed as given from the precommit hooks in checkboxes, and tick after tested them.
- [ ] IPInfoTest

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](https://github.com/demisto/content/edit/ipinfo-batch/Integrations/integration-Ipinfo.yml?pr=%2Fdemisto%2Fcontent%2Fpull%2F4699)
- [ ] [CHANGELOG](https://github.com/demisto/content/edit/ipinfo-batch/Integrations/integration-Ipinfo_CHANGELOG.md?pr=%2Fdemisto%2Fcontent%2Fpull%2F4699)

### CODE REVIEW

File: Integrations/integration-Ipinfo.yml [hunk 1]
```diff
@@ -34,30 +39,63 @@ script:
...
+                var reply = [];
+                var i = 0;
+                while (i < ips.length) {
+                    var batch = ips.slice(i, i + batch_size);
+                    var res = sendRequest(base + 'batch' + token, 'POST', true, JSON.stringify(batch));
+                    for (var property in res) {
+                        var ip_info = res[property];
+                        var ec = {IP: {Address: ip_info.ip, Hostname: ip_info.hostname, ASN: ip_info.org, Geo: {
+                            Location: ip_info.loc, Country: ip_info.country, Description: ip_info.city + ', ' + ip_info.region + ', ' + ip_info.postal + ', ' + ip_info.country}}};
+                        ec.DBotScore = {Indicator: property, Type: 'ip', Vendor: 'ipinfo', Score: 0};
```
Comment (Itay4) [inline Integrations/integration-Ipinfo.yml]: looks like we already have a function to do that for one, lets generalise this

**Comment by avidan-H** [comment]
@kirbles19 The DBotScore output descriptions are taken directly from our code conventions so only the changelog is actually relevant.

**Comment by kirbles19** [comment]
@avidan-H Done.

**Comment by avidan-H** [comment]
Test and merge dependent on this [issue](https://github.com/demisto/etc/issues/19767) @HemaHall

**Comment by Itay4** [comment]
closing as this is hanging for too long

**Pull request closed without merging**